packages/odyssey-react/src/components/Box/Box.theme.ts

```diff
@@ -13,16 +13,9 @@
 import type { ThemeReducer } from "@okta/odyssey-react-theme";
 
 export const theme: ThemeReducer = (theme) => ({
-  // REMOVE THIS
   /* eslint-disable @okta/odyssey/no-invalid-theme-properties */
-
-  // Font
-  FontFamilyBase: theme.FontFamilyBase,
-  FontWeightNormal: theme.FontWeightNormal,
-  FontStyleNormal: theme.FontStyleNormal,
-  FontSize: theme.FontSizeBody,
-  FontLineHeightBase: theme.FontLineHeightBase,
-  ColorTextBody: theme.ColorTextBody,
+  // Width/Height
+  MaxLineLength: theme.FontLineLengthMax,
 
   // Space
   SpaceRemXs: theme.SpaceRemXs,
```

Comment:
The props and classes associated with these spacing tokens target the *value* rather than the *property*. As such, I've left them in place here.

Reply:
I'm not sure I follow, can you explain further?

Reply:
Here's a large group from `box-margin`:

```scss
.marginTopXs {
  margin-block-start: var(--SpaceRemXs);
}

.marginTopS {
  margin-block-start: var(--SpaceRemS);
}

.marginTopM {
  margin-block-start: var(--SpaceRemM);
}

.marginTopL {
  margin-block-start: var(--SpaceRemL);
}

.marginTopXl {
  margin-block-start: var(--SpaceRemXl);
}

.marginRightXs {
  margin-inline-end: var(--SpaceRemXs);
}

.marginRightS {
  margin-inline-end: var(--SpaceRemS);
}

.marginRightM {
  margin-inline-end: var(--SpaceRemM);
}

.marginRightL {
  margin-inline-end: var(--SpaceRemL);
}

.marginRightXl {
  margin-inline-end: var(--SpaceRemXl);
}

.marginBottomXs {
  margin-block-end: var(--SpaceRemXs);
}

.marginBottomS {
  margin-block-end: var(--SpaceRemS);
}

.marginBottomM {
  margin-block-end: var(--SpaceRemM);
}

.marginBottomL {
  margin-block-end: var(--SpaceRemL);
}

.marginBottomXl {
  margin-block-end: var(--SpaceRemXl);
}

.marginLeftXs {
  margin-inline-start: var(--SpaceRemXs);
}

.marginLeftS {
  margin-inline-start: var(--SpaceRemS);
}

.marginLeftM {
  margin-inline-start: var(--SpaceRemM);
}

.marginLeftL {
  margin-inline-start: var(--SpaceRemL);
}

.marginLeftXl {
  margin-inline-start: var(--SpaceRemXl);
}
```

These map to t-shirt size `spacing` types that are unaware of their property (margin/padding) or position (inline/block). Regardless of the associated property, the application of the Spacing unit itself is expected to remain consistent. We could create bespoke tokens for every property application throughout Box; however, if they are themed individually, the props will no longer function as intended.

Reply:
Something like `XlSpacing` might match our current naming structure better, but `MarginInlineStartXL` feels iffy to surface.

Reply:
@edburyenegren-okta I understand now thank you, yea this works fine for now, even if the intended result is a bit harder to follow for users of the API. They'll need to understand that the value is shared and applies in a few different locations but this works for now yes!